### PR TITLE
Bump version, set release names

### PR DESCRIPTION
**Pull request by FriederikeHanssen**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/sarek/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/sarek _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -52,9 +54,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 | `gatk4spark/baserecalibrator` | `GATK4_BASERECALIBRATORSPARK` | `GATK4SPARK_BASERECALIBRATOR` |
 | `gatk4spark/markduplicates`   | `GATK4_MARKDUPLICATESSPARK`   | `GATK4SPARK_MARKDUPLICATES`   |
 
-## [3.3.2](https://github.com/nf-core/sarek/releases/tag/3.3.2) - Ráhpajávvre
+## [3.3.2](https://github.com/nf-core/sarek/releases/tag/3.3.2) - Ráhpaädno
 
-Ráhpajávvre is the Lule Sámi spelling of Rapaselet.
+Ráhpaädno is the Lule Sámi spelling of Rapaälven.
 
 ### Added
```
Comment (maxulysse) [inline CHANGELOG.md:61]: why the change?

Comment (FriederikeHanssen) [reply CHANGELOG.md]: bad copy paste on my part

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev
+## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkkå
```
Comment (maxulysse) [inline CHANGELOG.md:8]:
```suggestion
## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkko
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
...
+
+Pårtetjåkkå is mountain in the south of the park.
```
Comment (maxulysse) [inline CHANGELOG.md:10]:
```suggestion
Pårtetjåkko is mountain in the south of the park.
```

Comment (FriederikeHanssen) [inline CHANGELOG.md:10]:
```suggestion
Pårtetjåkkå is a mountain in the south of the park.
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev
+## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkko
+
+Pårtetjåkkå is a mountain in the south of the park.
```
Comment (maxulysse) [inline CHANGELOG.md:10]:
```suggestion
Pårtetjåkko is a mountain in the south of the park.
```

Comment (FriederikeHanssen) [inline CHANGELOG.md:10]:
```suggestion
Pårtetjåkko is a mountain in the south of the park and the Lule Sámi spelling of Bårddetjåhkkå.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,7 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## dev
+## [3.4](https://github.com/nf-core/sarek/releases/tag/3.4) - Pårtetjåkko
```
Comment (maxulysse) [inline CHANGELOG.md:8]: `3.4` or `3.4.0`?

Comment (FriederikeHanssen) [reply CHANGELOG.md:8]: we  have been going back and forth on this. 3.0, 3.1, 3.2.0, 3.30 happy to be consistent there.

Comment (maxulysse) [reply CHANGELOG.md:8]: Let's try to follow the trend we established since `3.2.0`.
But I agree, for me `3.4.0` is the same as `3.4`

**Review by asp8200 — APPROVED** [review]
dohkkeheapmi
